src/crate/client/sqlalchemy/dialect.py

```diff
@@ -261,7 +262,7 @@ def get_pk_constraint(self, engine, table_name, schema=None, **kw):
 
             def result_fun(result):
                 rows = result.fetchall()
-                return set(map(lambda el: el[0], rows))
+                return list(set(map(lambda el: el[0], rows)))
```

Comment:
While I don't know about the specific background why there has been a `set` used here, can you elaborate why this should now return a list?

Reply:
Dear @Aymaru,

apologies for the late reply. We've split off this amendment into #426. For you, it apparently solved this issue:

> Changing the return type from set to list on get_pk_constraint(), solves a compatibility issue from Apache Superset that doesn't show the metadata from CrateDB tables in the SQL Lab.

Maybe due to the upgrade to SQLAlchemy 1.4, and maybe other changes in Apache Superset, this is not needed anymore?

With kind regards,
Andreas.